flake.nix

```
@@ -0,0 +1,47 @@
+{
+  description = "Flake do fortes-webservice-wrapper";
+
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+
+        pkgsAllowUnfree = import nixpkgs {
+          system = "x86_64-linux";
+          config = { allowUnfree = true; };
+        };
+
+        config = {
+          projectDir = ./.;
+        };
+
+      in
+      {
+
+        poetryEnv = import ./mkPoetryEnv.nix.nix {
+          pkgs = nixpkgs.legacyPackages.${system};
+        };
+
+        env = pkgsAllowUnfree.poetry2nix.mkPoetryEnv config;
+
+        devShell = pkgsAllowUnfree.mkShell {
+          buildInputs = with pkgsAllowUnfree; [
+            #(pkgsAllowUnfree.poetry2nix.mkPoetryEnv config)
```

Por hora isso pode ser removido pois não funciona nem é usado.

```suggestion
        devShell = pkgsAllowUnfree.mkShell {
          buildInputs = with pkgsAllowUnfree; [
```